crates/completion/src/tests.rs

```diff
@@ -2134,3 +2144,103 @@ fn issue_885() {
         "#]],
     );
 }
+
+#[test]
+fn test_custom_label_prefix_ref() {
+    let mut config = SyntaxConfig::default();
+    config.label_definition_commands.insert("asm".to_string());
+    config.label_reference_commands.insert("asmref".to_string());
+    config
```

Comment:
I think you are missing `config.label_definition_prefixes` here. If you add 
```rs 
config.label_definition_prefixes.insert("asm".to_string(), "asm:".to_string());
```
then the test passes. Shouldn't it be like this config?

```json
{ 
  "labelDefinitionPrefixes": { "ams": "ams:" },
  "labelReferencePrefixes": { "amsref": "ams:" }
}
```